QuestionServlet.java

```diff
@@ -0,0 +1,50 @@
+
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class QuestionServlet
+ */
+@WebServlet("/question")
+public class QuestionServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public QuestionServlet() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		// TODO Auto-generated method stub
+
+		String test = Quizz.Hello();
+		
+		request.setAttribute( "sentence", test );
```

Comment:
je vous recommande de mettre explicitement `this.` sur les appels de méthodes, pour des questions de lisibilité, mais c'est pas bloquant :)